### ISSUE TITLE
Some dtypes cannot be deserialized

### ISSUE THREAD

**Issue by cchudant** [mithril-security/bastionlab#185]
```
File ~/bastionlab/client/src/bastionlab/polars/remote_polars.py:959, in <listcomp>(.0)
    955 @staticmethod
    956 def _from_reference(client: BastionLabPolars, ref: ReferenceResponse) -> LDF:
    957     header = json.loads(ref.header)["inner"]
    958     df = pl.DataFrame(
--> 959         [pl.Series(k, dtype=getattr(pl, v)()) for k, v in header.items()]
    960     )
    961     return FetchableLazyFrame(
    962         _identifier=ref.identifier,
    963         _inner=df.lazy(),
    964         _meta=Metadata(client, [EntryPointPlanSegment(ref.identifier)]),
    965     )

TypeError: getattr(): attribute name must be string
```
This is the case for List, Datetime and probably others.
If we don't want to support things like List, I think it's ok; but the error message could be clearer

Reproduction: fetch a dataframe with type List[i64] or Datetime

**Comment by kbamponsem** [comment]
This has already been solved by the new data conversion policy.
It will be merged soon.

Link to the fix:
https://github.com/mithril-security/bastionlab/blob/ML_notebook/client/src/bastionlab/polars/remote_polars.py#L1020

**Comment by cchudant** [comment]
@kbamponsem 
Oh cool!
Please put the words `closes #185` in the comments when you create your PR, this will tell github to automatically close this issue when it gets merged :)
